### PR TITLE
translated 2 new lines. by Benny

### PR DESCRIPTION
**Pull request by BennySchindelhauer**
hope thats correct this way.

#: index.php:1111
msgid ""
"With PalMA, you can share your documents and your desktop\n"
"            with your learning group."
msgstr ""
"Mit PalMA kanst du deine Dokumente und deinen Desktop\n"
"			mit deiner Lerngruppe teilen."

#: index.php:1113
msgid ""
"Team members can join the group at any time. All they need\n"
"            is URL and PIN."
msgstr ""
"Temmitglieder können der Gruppe jederzeit beitreten. Sie\n"
"			brauchen nur die URL und den PIN."

### CODE REVIEW

**Comment by stweil** [comment]
Hello Benny,

technically, your pull request is correct, but you might review your translation and also your commit message. We also require an author name before we pull any request. Please have a look on other PalMA commits to see how it is done.

Cheers
Stefan

**Comment by kba** [comment]
@BennySchindelhauer:

Please make sure that you set up the git username and email adress for git:
```
git config --global user.name "Benny Schindelhauer"
git config --global user.email "...@uni-mannheim.de"
```

Then clone your fork again into a new directory, cd to it, create a new branch:
```
git checkout -b 'en-translation'
```

Copy over your changes, add them, commit with a message, then push your branch over:

```
git push origin en-translation
```

And then open a new pull request that we can merge.

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po [hunk 1]
```diff
@@ -164,12 +164,16 @@ msgid ""
...
 msgstr ""
+"Mit PalMA kanst du deine Dokumente und deinen Desktop\n"
+"			mit deiner Lerngruppe teilen."
 
 #: index.php:1113
 msgid ""
 "Team members can join the group at any time. All they need\n"
 "            is URL and PIN."
 msgstr ""
+"Temmitglieder können der Gruppe jederzeit beitreten. Sie\n"
```
Comment (stweil) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: s/Temmitglieder/Teammitglieder/

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po [hunk 2]
```diff
@@ -164,12 +164,16 @@ msgid ""
 "With PalMA, you can share your documents and your desktop\n"
 "            with your learning group."
 msgstr ""
+"Mit PalMA kanst du deine Dokumente und deinen Desktop\n"
```
Comment (stweil) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: s/kanst/kannst/

Comment (stweil) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: Should we use "du" or "Sie"? We already have "Sie wurden abgemeldet".

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po [hunk 3]
```diff
@@ -164,12 +164,16 @@ msgid ""
...
+"			mit deiner Lerngruppe teilen."
 
 #: index.php:1113
 msgid ""
 "Team members can join the group at any time. All they need\n"
 "            is URL and PIN."
 msgstr ""
+"Temmitglieder können der Gruppe jederzeit beitreten. Sie\n"
+"			brauchen nur die URL und den PIN."
```
Comment (stweil) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: PIN = Persönliche Identifikationsnummer, so we need "die PIN" instead of "den PIN"

**Comment by zuphilip** [comment]
Superseded by https://github.com/UB-Mannheim/PalMA/commit/e30ff6dff6a154fa4afdcca00d5c4c7353714123

**Pull request closed without merging**